packages/core/src/config/__tests__/config.test.ts

```diff
@@ -239,3 +241,44 @@ describe('generation ignore object', () => {
     expect(config).toMatchSnapshot();
   });
 });
+
+describe('rootRedoclyConfigSchema synchronization', () => {
```

Comment:
I'm reluctant to add a unit test for this here. Imagine you just want to add a property--but you wouldn't be able to do it without releasing a new config version. Can you move it to something like a smoke test (so it's not required for merging PRs)?